setup.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+from os import path
+from setuptools import setup, find_packages
+
+
+pkg_name = "intanrhdreader"
+here = path.abspath(path.dirname(__file__))
+
+with open(path.join(here, "README.md"), "r") as f:
+    long_description = f.read()
+
+with open(path.join(here, "requirements.txt")) as f:
+    requirements = f.read().splitlines()
+
+with open(path.join(here, "version.py")) as f:
```

Comment:
why not import `version.py`?

Reply:
I am not sure.  DataJoint Python and the Elements are all configured this way.